README.md

```diff
@@ -19,26 +19,24 @@
 
 ---
 
-## Development
+## Build docs locally
 
-1. Checkout this repository
-1. Install mintlify with `npm i -g mintlify@latest`
-1. Run `mintlify dev`
+1. Fork this repository.
+1. Install mintlify with `npm i -g mintlify@latest`.
+1. Run `mintlify dev`.
 
 ## Deploying 
 
-To build the final product, we have a couple of additional steps:
+To build the final product:
 
-1. Build the JS Protocol Docs
+1. Build the JS Protocol Docs:
 
 - `npm install -g jsdoc-to-markdown`
 - `./bin/jsdocs.sh`
 
-2. Build the PDCP API reference documentation
+2. Build the PDCP API reference documentation:
```

Comment:
Remember, whoever is running this might be new to the project. Tell them what PDCP API documentation is, preferably in the section about actually working on docs, which does not yet exist.